### PR TITLE
Add WMT workload variants

### PR DESCRIPTION
**Pull request by runame**
I haven't verified yet that the modeldiff tests pass since even the one for the standard WMT workload currently fails with my setup. I won't have time to work on this until tomorrow, so I decided to already create this PR in case someone wants to verify the tests themselves.

I think the test for the GLU variant might have to be modified since a new dense layer is added.

### CODE REVIEW

**Comment by priyakasimbeg** [comment]
Hi Runa, just checking yet if you were able to verify whether the modeldiff tests are passing.

**Comment by runame** [comment]
Sorry for the delay! All modeldiff tests pass now and this PR is ready to be reviewed.

File: algorithmic_efficiency/workloads/wmt/wmt_jax/models.py [hunk 1]
```diff
@@ -192,15 +204,16 @@ def __call__(self, inputs, encoder_mask=None):
       output after transformer encoder block.
     """
     cfg = self.config
+    norm_first = cfg.norm_first
```
Comment (priyakasimbeg) [inline algorithmic_efficiency/workloads/wmt/wmt_jax/models.py:207]: Nit: can we change 'norm_first' to 'pre_ln', I think 'pre_ln' and 'post_ln' are the terminology we've been using.

File: algorithmic_efficiency/workloads/wmt/wmt_pytorch/models.py [hunk 2]
```diff
@@ -106,7 +106,11 @@ def __init__(self,
                nlayers: int = 6,
                dropout_rate: Optional[float] = 0.1,
                attention_dropout_rate: Optional[float] = 0.1,
-               layer_norm_eps: float = 1e-6):
+               activation: Union[str, Callable[[Tensor], Tensor]] = F.relu,
+               glu: bool = False,
+               layer_norm_eps: float = 1e-6,
+               attention_temp: float = 1.0,
+               norm_first: bool = True):
```
Comment (priyakasimbeg) [inline algorithmic_efficiency/workloads/wmt/wmt_pytorch/models.py]: Same nit as above; can we change 'norm_fist' -> 'pre_ln'

File: algorithmic_efficiency/workloads/wmt/wmt_jax/models.py (hunk 1 above)
Comment (runame) [reply algorithmic_efficiency/workloads/wmt/wmt_jax/models.py]: Done.

**Review by priyakasimbeg — APPROVED** [review]

File: algorithmic_efficiency/workloads/wmt/wmt_pytorch/models.py (hunk 2 above)
Comment (runame) [reply algorithmic_efficiency/workloads/wmt/wmt_pytorch/models.py]: Already renamed everywhere in the repo.

**Review by priyakasimbeg — APPROVED** [review]

**Pull request merged**